packages/api/internal/handlers/sandbox_create.go

```diff
@@ -156,13 +159,37 @@ func (a *APIStore) PostSandboxes(c *gin.Context) {
 		envdAccessToken = &accessToken
 	}
 
+	var maskRequestHost *string = nil
+	if body.Network != nil && body.Network.MaskRequestHost != nil {
```

Comment:
wouldn't it make sense to do it inside the already present `if body.Network != nil` block?